source/includes/_custom_screeners.md

```diff
@@ -0,0 +1,197 @@
+question_text | String | This is the question text that will be served to our respondents.
+answer_type | Integer | This value will determine how campaign_qualifications pre-codes are generated for quotas. See **Answer Types** section for more information.
+country_language_id | Integer | Country language targeting for the qualification.
+en_translation | String | The English translation of the question.
+
+## Create a custom qualification answer
+
+Create a custom qualification answer to correspond with your custom qualifications.
+
+> Sample Request Payload - Update custom qualification name
+
+```json
+{
+    "option_text": "Corporate Chef"
+}
+```
+
+> Sample Response 
+
+```json
+{
+    "option_text": "Corporate Chef",
+    "pre_code": 4,
+    "en_translation": "Corporate Chef",
+    "country_language_id": 1
+}
+```
+
+### HTTP Request
+
+`POST https://www.tapresearch.com/api/v1/custom_qualifications/:question_id/custom_qualification_answers`
+
+
+### Required Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+option_text | String | The answer text that is shown to our respondents.
+
+### Response
+Parameter | Type | Description
+--------- | ---- | -----------
+option_text | String | The answer text that is shown to our respondents.
+pre_code | Integer | This is the value(s) you will be passing into the campaign_qualifications pre_codes array to describe which respondents qualify for a survey.
+en_translation | String | The English translation of the question.
+country_language_id | Integer | Country language targeting for the qualification.
+
+## Create a campaign qualification
+
+Create a campaign qualification which will allow you to associate your custom qualifications and answers to a specific campaign.
+
+> Sample Request Payload - 
+
+```json
+{
+    "question_id": 100112713,
+    "pre_code_values": "1"
+}
+```
+
+> Sample Response 
+
+```json
+{
+    "id": 50394692475,
+    "question_id": 100112713,
+    "pre_codes": [
+        1
+    ]
+}
+```
+
+### HTTP Request
+
+`POST https://www.tapresearch.com/api/v1/campaigns/:campaign_id/campaign_qualifications`
+
+
+### Required Parameters
+Parameter | Type | Description
+--------- | ---- | -----------
+id | Integer | This the unique identifier of the campaign qualification.You will need to use this parameter to update a specific campaign qualification.
+question_id | Integer | This value will be used to fetch qualification details.
+pre_code_values | String | The pre_code value received in the response of returned when creating a custom qualification answer. See **Create a custom qualification answer** for more information.
+
+
+### Response
+Parameter | Type | Description
+--------- | ---- | -----------
+question_id | Integer | This value will be used to fetch qualification details.
+pre_codes | String | Values to be passed in the pre_codes array to describe which respondents qualify for a survey.
```

Comment:
It looks like the response here is an array of `int`'s, right? 
https://github.com/TapResearch/slate/pull/5/files#diff-c9d1c5474bc24df142fcac5764bab9930c429f049932b93ad34d0e9c805e1287R123

Reply:
Fixed!